.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Lines starting with '#' are comments.
```

Comment:
I would start with a summary describing what the purposed of this file is, followed by a link to the documentation.

Reply:
Addressed in 6802770.